.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: node_js
-node_js: "8"
+node_js:
+- node
+- lts/*
+- 8
+services:
+  - mongodb
```

Comment:
Nice! I didn't know that this was enough for enabling a mongo connection in test environment! Good solution!

Reply:
Yeah, it's great! There's a bunch of other services as well, like redis and other databases